src/main/java/org/apache/sling/feature/cpconverter/cli/ShutDownHook.java

```diff
@@ -76,4 +82,15 @@ public void run() {
         logger.info("+-----------------------------------------------------+");
     }
 
+    private void cleanUp(){
+        String tmpDir = System.getProperty("java.io.tmpdir");
+        logger.info("Cleaning up tmp directories {}, {}", tmpDir + "sub-content-packages", tmpDir + "syntethic-content-packages" );
+
+        try {
+            FileUtils.deleteDirectory( new File (tmpDir, "syntethic-content-packages") );
+            FileUtils.deleteDirectory( new File(tmpDir, "sub-content-packages") );
+        } catch (IOException e) {
+            logger.error( "Error Deleting {}, {}", tmpDir + "sub-content-packages", tmpDir + "syntethic-content-packages");
+        }
```

Comment:
I would make `"sub-content-packages"` and `"syntethic-content-packages"` constants...

Reply:
@bosschaert this was conflicting for me, I _was_ going to make them constants, but, the existing classes don't have them as constants, and I decided to follow the same practice for consistency sake. Maybe another issue to track making a common constants class across the board?

Reply:
Ok fair enough. No problem then :)